distribution/docs/src/main/resources/content/_coreConcepts/high-availability-supported-capabilities.adoc

```diff
@@ -0,0 +1,29 @@
+:type: subCoreConcept
+:status: published
+:title: High Availability Supported Capabilities
+:parent: Introduction to High Availability
+:order: 00
+
+===== {title}
+
+Only these capabilities are supported in a Highly Available Cluster. For an even more detailed view, see the <<_high_availability_supported_features,List of Supported Features>>.
+
+* User Interfaces:
+** Simple
+** Standard
```

Comment:
❗️ ❓  Do we have to support the standard UI? I thought we didn't due to the CometD endpoint being real tricky to get right in an HA setup.